From randy.dunlap@oracle.com  Thu Feb 18 08:05:57 2010
From: Randy Dunlap <randy.dunlap@oracle.com>
Date: Tue, 16 Feb 2010 14:59:07 -0800
Subject: Staging: rtl8187se: needs semaphore.h
To: Stephen Rothwell <sfr@canb.auug.org.au>
Cc: Greg KH <gregkh@suse.de>, devel@driverdev.osuosl.org
Message-ID: <4B7B233B.9030601@oracle.com>


From: Randy Dunlap <randy.dunlap@oracle.com>

rtl8187se needs to include semaphore.h to prevent build errors:

drivers/staging/rtl8187se/ieee80211/ieee80211.h:1004: error: field 'wx_sem' has incomplete type
drivers/staging/rtl8187se/ieee80211/ieee80211.h:1005: error: field 'scan_sem' has incomplete type

Signed-off-by: Randy Dunlap <randy.dunlap@oracle.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/rtl8187se/ieee80211/ieee80211.h |    1 +
 1 file changed, 1 insertion(+)

--- a/drivers/staging/rtl8187se/ieee80211/ieee80211.h
+++ b/drivers/staging/rtl8187se/ieee80211/ieee80211.h
@@ -29,6 +29,7 @@
 #include <linux/jiffies.h>
 #include <linux/timer.h>
 #include <linux/sched.h>
+#include <linux/semaphore.h>
 #include <linux/wireless.h>
 #include <linux/ieee80211.h>
 
